pkg/syncer/cnsoperator/controller/cnsfileaccessconfig/cnsfileaccessconfig_controller.go

```diff
@@ -369,7 +371,7 @@ func (r *ReconcileCnsFileAccessConfig) Reconcile(ctx context.Context,
 			msg := fmt.Sprintf("CNS Volume: %s is not RWX volume", volumeID)
 			err = logger.LogNewError(log, msg)
 			setInstanceError(ctx, r, instance, msg)
-			return reconcile.Result{RequeueAfter: timeout}, err
+			return reconcile.Result{RequeueAfter: timeout}, reconcile.TerminalError(err)
```

Comment:
Changing this to a terminal error so that the reconcile does not get retried for such volumes